dev/Scroller/TestUI/ScrollerDynamicPage.xaml.cs

```diff
@@ -2008,7 +2008,94 @@ private void BtnAddRepeatedSnapPoint_Click(object sender, RoutedEventArgs e)
             }
         }
 
-        private void BtnClearSnapPoints_Click(object sender, RoutedEventArgs e)
+        private void BtnChangeView_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                double? horizontalOffset = string.IsNullOrWhiteSpace(txtCOAHO.Text) ? (double?)null : Convert.ToDouble(txtCOAHO.Text);
```

Comment:
>txtCOAHO [](start = 69, length = 8)

What does the COAHO stand for ? The acronyms don't seem intuitive..

Reply:
I renamed a few control names. That one was outdated anyways: COAHO for ChangeOffsetAsyncHorizontalOffset.